src/sentry/conf/server.py

```diff
@@ -845,6 +845,8 @@ def create_partitioned_queues(name):
     "organizations:integrations-event-hooks": False,
     # Enable data forwarding functionality for organizations.
     "organizations:data-forwarding": True,
+    # Enable experimental performance improvements.
+    "organizations:enterprise-perf": False,
```

Comment:
Are we adding as a feature flag because we suspect we may cause regressions with attempts at making things faster?

Reply:
That's a side benefit, the main goal is to create A/B segments to measure how much our changes improve performance relative to a control group.